src/dcd_mapping/lookup.py

```diff
@@ -255,14 +286,28 @@ def _get_normalized_gene_response(
 
     # try taking the first word in the target name
     if metadata.target_gene_name:
-        parsed_name = metadata.target_gene_name.split(" ")[0]
-        gene_descriptor = _normalize_gene(parsed_name)
-        if gene_descriptor:
-            return gene_descriptor
+        parsed_name = ""
+        if "_" in metadata.target_gene_name:
+            parsed_name = metadata.target_gene_name.split("_")
+        if " " in metadata.target_gene_name:
+            parsed_name = metadata.target_gene_name.split(" ")
+        for word in parsed_name:
+            word = extract_potential_gene_symbol(word)
+            gene_descriptor = _normalize_gene(word)
+            if gene_descriptor:
+                return gene_descriptor
 
     return None
 
 
+def extract_potential_gene_symbol(word: str) -> str:
+    """Remove paranetheses from a potential gene symbol
```

Comment:
blank line between the first sentence in the docstring and the first :param: line